easybuild/easyblocks/o/openfoam.py

```diff
@@ -281,6 +281,9 @@ def build_step(self):
                 "Cleaning .*",
             ]
             run_cmd_qa(cmd_tmpl % 'Allwmake.firstInstall', qa, no_qa=noqa, log_all=True, simple=True)
+        elif LooseVersion(self.version) >= LooseVersion('1600'):
```

Comment:
It was added in 1606. How would you clean this up? (Too much of the package looks like cut-n-paste already).